index.html

```diff
@@ -25,10 +25,14 @@
         <li class="nav-item-two">About</li>
       </ul>
       <h1>Kyle Hillman Portfolio</h1>
-      <section class="projects">
-        <img />
-        <img />
-        <img />
+      <!-- Display projects template -->
+      <section id="projects" class="projects">
+        <summary class="template">
+          <img />
+          <address><a href="">Project Name</a></address>
+          <time>Date published</time>
+          <div class="project-description">Project Description</div>
+        </summary>
       </section>
       <div class="skills">
```

Comment:
Change this `div` to a `section`. Do the same with the `div` around your About section. This will make your code more semantically readable, as well as making it easier to show/hide sections later.